tests/platforms/linux/system/test_mixin__verify_python.py

```diff
@@ -32,10 +32,7 @@ def test_match(create_command, first_app_config, capsys):
         [
             "python3",
             "-c",
-            (
-                "import sys; "
-                "print(f'{sys.version_info.major}.{sys.version_info.minor}')"
-            ),
+            ("import sys; print(f'{sys.version_info.major}.{sys.version_info.minor}')"),
```

Comment:
Just wanted to check if you're okay with this on a single line, or if you want me to try to break it up with a `\`.  I'll do this for all command-like lines that got joined back up by ruff.